hack/rpm-deps.sh

```diff
@@ -105,6 +111,28 @@ bazel run \
     $libvirtdevel_base \
     $libvirtdevel_extra
 
+bazel run \
+    --config=${ARCHITECTURE} \
+    //:bazeldnf -- rpmtree \
+    --public --nobest \
+    --name sandboxroot_x86_64 \
+    --repofile rpm/centos-repo.yaml \
+    --basesystem centos-stream-release \
+    $centos_base \
+    $centos_extra \
+    $sandbox_base
+
+bazel run \
+    --config=${ARCHITECTURE} \
+    //:bazeldnf -- rpmtree \
+    --public --nobest \
+    --name sandboxroot_aarch64 --arch aarch64 \
+    --repofile rpm/centos-repo.yaml \
+    --basesystem centos-stream-release \
+    $centos_base \
+    $centos_extra \
+    $sandbox_base
```

Comment:
Nit: these two rules should not appear in between two rules that differ only in architecture (`libvirt-devel_x86_64` and `libvirt-devel_aarch64`).

Another nit: to keep things nice, tidy and consistent with the rest of the file, you should define `sandbox_base` right before these two rules.

Reply:
Done.